src/desktop/apps/artist/view_helpers.coffee

```diff
@@ -10,6 +10,11 @@ showHelpers = require '../../components/show_cell/helpers.coffee'
 module.exports =
   capitalize: capitalize
   showHelpers: showHelpers
+  
+  # TODO:: ARTIST_MARKET_DATA_TEST remove after test closes
```

Comment:
@mzikherman - to your main comment about A/B test - I was thinking that I can do all the code changes and since the rendering of the new stuff is completely isolated and the rest is just showing/hiding existing components - I can do a feature check here and assign proper defaults otherwise. And if the feature is enabled - do the test group check.

The alternative is doing this feature check in several places (since templates rendered both on the server and through js) and eventually do a complete replace of that code with split test checks....

Do you see issues with that approach?

Reply:
Not at all, whatever's easier! That sounds great.

You're right in pointing out that replacing conditionals/gating with the proper AB test check might need to happen in a bunch of places as the last step, and in various templates and coffeescript/js files, etc.

So to avoid having to do that (and the potential for bugs creeping thru at the end), I would definitely agree with your plan to do that check in one place if possible, so that's the only thing that changes when the AB test is ready to launch. Which you're doing in the server-side artist page routes file, so I think we're all 👌